packages/cli/src/commands/create/monorepo.ts

```diff
@@ -84,12 +84,37 @@ export const createMonorepo: CreateMonorepoFunc = async ({
     copyFileSync(file, destFilePath)
   })
 
-  console.log(bold(cyan('[info] Running `yarn install`...')))
-  await runCommand(`yarn install`, { cwd: repoDir })
+  if (packageManager === 'yarn' || packageManager === 'pnpm') {
+    const json = JSON.parse(
```

Comment:
can we maybe abstract this part into a separate function because on the first glimpse its a bit difficult to read

Reply:
thanks for the feedback! https://github.com/storyblok/field-plugin/pull/221/commits/ff656f8409b96d6787fb62be680f196b495e52de